webapp/channels/src/selectors/preferences.ts

```diff
@@ -15,3 +15,21 @@ export const arePreviewsCollapsed = (state: GlobalState) => {
         Preferences.COLLAPSE_DISPLAY_DEFAULT !== 'false',
     );
 };
+
+export const isSendOnCtrlEnter = (state: GlobalState) => {
```

Comment:
Are we getting these values using `getBoolPreference` anywhere else? If so, could you either change those now or file a ticket to do that later?

We might also need to move these into mattermost-redux to do that

Reply:
Currently use use the generic `getPreference` selector from redux store to get some of these values, and for some newer settings we have these functional selectors outside the `mattermost-redux` package.

Isn't the idea to get rid of the mattermost-redux package and add new redux stuff in `channels/src/selectors/`?

Reply:
I mostly dislike the way that plugins use mattermost-redux, but I'm undecided on the package itself in the web app. The only reason I mentioned it here is in case something else in mattermost-redux needs this because it can't import anything from outside of itself

Reply:
No, nothing in the redux package uses this.